From 832a3cb827f74f8d5a0d37d240a9dae291212f01 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Thu, 24 Oct 2013 13:18:06 +0200
Subject: [PATCH] Recognize more ARM architectures

Gotten via rpm --eval %arm.
Not reported upstream, since it's not clear how should one submit a patch...
---
 kBuild/env.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kBuild/env.sh b/kBuild/env.sh
index fa3f7be..0ea9ddb 100755
--- a/kBuild/env.sh
+++ b/kBuild/env.sh
@@ -358,7 +358,7 @@ if test -z "$KBUILD_HOST_ARCH"; then
         hppa64|parisc64)
             KBUILD_HOST_ARCH='hppa64'
             ;;
-        arm|armv4l|armv5tel|armv5tejl)
+	armv3l|armv4b|armv4l|armv4tl|armv5tel|armv5tejl|armv6l|armv7l|armv7hl|armv7hnl)
             KBUILD_HOST_ARCH='arm'
             ;;
         alpha)
-- 
1.8.3.1

